doc/introduction.rst

```diff
@@ -1,9 +1,9 @@
 Introduction
 ############
 
-`nmigen` is a Python-based tool that aims at automating further the VLSI design process.
+`nMigen` is a Python-based tool that aims at automating further the VLSI design process.
```

Comment:
No no, it shouldn't be set in monospace either. Just nMigen, without <code>\`</code>.